README.md

```diff
@@ -82,6 +82,20 @@ WARNING! this will erase the catalog, and thus require re-seeding from scratch.
   Basically an especially inconvenient confirmation dialogue.  For safety's sake, the full command that skips that warning can be constructed by the user as needed, so as to prevent unintentional copy/paste dismissal when the user might be administering multiple deployment environments simultaneously.  Inadvertent database wipes are no fun.
   * `db:reset` will make sure db is migrated and seeded.  If you want to be extra sure: `RAILS_ENV=[environment] bundle exec rake db:migrate db:seed`
 
+#### Drop or populate the catalog for a single endpoint
+
+There are rake tasks for clearing out all database entries for a single endpoint. To run either of the rake tasks, invoke them as follows, prepending the `RAILS_ENV` and giving the name of the endpoint as an argument (note that the argument is not a string):
+
+```ruby
+RAILS_ENV=production bundle exec rake drop[services-disk01]
+```
+
+will clear out all database entries for the single endpoint name specified. The following will populate the database for the single endpoint name specified:
```

Comment:
I would prefer to have the "what will it do" in the same position for each of these rake task explanations - preferably before.  I don't want to read it;  I want to skim it.  I might even have a `#####` heading for each task?

Reply:
View #281